.travis.yml

```diff
@@ -20,13 +20,20 @@ php:
 matrix:
     include:
         - php: 7.2
-          env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable' SYMFONY_DEPRECATIONS_HELPER=weak
+          env:
+              - COMPOSER_FLAGS='--prefer-lowest --prefer-stable'
+              - SYMFONY_DEPRECATIONS_HELPER=weak
         - php: 7.3
-          env: DEPENDENCIES='dev'
-          env: SYMFONY_DEPRECATIONS_HELPER='max[self]=0'
+          env:
+              - DEPENDENCIES='dev'
+              - SYMFONY_DEPRECATIONS_HELPER='max[self]=0'
         # Test LTS version
         - php: 7.3
           env: SYMFONY_VERSION='^3'
+        - php: 7.3
+          env: SYMFONY_VERSION='^4@dev'
+        - php: 7.3
+          env: SYMFONY_VERSION='^5@dev'
```

Comment:
You should use `- DEPENDENCIES='dev'` here, and remove it from line 28, where doesn't make sense.
Moreover, we don't need dev deps on Symfony 4 (I know there's 4.4-beta now, but stable will be out in a few days)

Reply:
> Moreover, we don't need dev deps on Symfony 4 (I know there's 4.4-beta now, but stable will be out in a few days)

Happy to go with just `^4`, but until (roughly) release this error will be thrown:
```
The requested package symfony/lts ^4 is satisfiable by symfony/lts[4.x-dev] but these conflict with your requirements or minimum-stability
```

Is that the prefered option?

Reply:
We need to remove symfony/lts, it's deprecated. To force Symfony version we can rely on requiring related `symfony/framework-bundle` package, or replace version with sed. For this last option, see https://github.com/PUGX/filter-bundle/blob/master/.travis.yml#L37

Reply:
After a couple of facepalm-worthy pushes, I think I have it now :smile_cat: